Makefile

```diff
@@ -430,7 +430,7 @@ kind-upgrade-chart: kind-load-image
 kind-install: kind-load-image
 	kubectl config use-context kind-kube-ovn
 	@$(MAKE) kind-untaint-control-plane
-	sed 's/VERSION=.*/VERSION=$(VERSION)/' dist/images/install.sh | bash
+	sed 's/VERSION=.*/VERSION=$(VERSION)/' dist/images/install.sh | ENABLE_TPROXY=true bash
```

Comment:
As tproxy is not enabled by default, we should pass this environment variable in GitHub actions instead of the Makefile:

[.github/workflows/build-x86-image.yamlL924](https://github.com/kubeovn/kube-ovn/blob/master/.github/workflows/build-x86-image.yaml#L923)
[.github/workflows/scheduled-e2e.yaml#L404](https://github.com/kubeovn/kube-ovn/blob/master/.github/workflows/scheduled-e2e.yaml#L404)

Reply:
when we use make kind-install and make e2e on local environment, the case will always fail

Reply:
What about moving the test cases out of `kube-ovn-conformance-e2e`, just like `ovn-vpc-nat-gw-conformance-e2e` does?

Reply:
This method is of course possible, but I feel that adding three github workerflows (ipv4, ipv6, dual) to this kind of case is a bit overkill.